reana_client/utils.py

```diff
@@ -102,7 +102,7 @@ def load_reana_spec(filepath, skip_validation=False):
                 get('specification')
             kwargs['parameters'] = \
                 reana_yaml.get('inputs', {}).get('parameters', {})
-
+        kwargs['original'] = True
```

Comment:
Note that CWL and Yadage workflow engine components need to be updated, otherwise they stop working:

```console
$ reana-dev run-example -c . -w yadage
[reana-demo-worldpopulation] reana-client create -f reana-yadage.yaml -n worldpopulation.yadage
Workflow could not be created: 
yadage_load() got an unexpected keyword argument 'original'
```

```console
$ reana-dev run-example -c . -w cwl                                                          1 ↵
[reana-demo-worldpopulation] reana-client create -f reana-cwl.yaml -n worldpopulation.cwl
Workflow could not be created: 
cwl_load() got an unexpected keyword argument 'original'
```

Serial works OK.